tests/integration/Owncloud/OcisPhpSdk/ResourceInviteTest.php

```diff
@@ -124,12 +162,38 @@ public function testInviteWithExpiry(): void
     {
         $tomorrow = new \DateTimeImmutable('tomorrow');
         $share = $this->fileToShare->invite($this->einstein, $this->viewerRole, $tomorrow);
-        $this->assertInstanceOf(\DateTimeImmutable::class, $share->getExpiration());
-        $this->assertSame($tomorrow->getTimestamp(), $share->getExpiration()->getTimestamp());
+        $shareResourceExpirationDateTime = $share->getExpiration();
+        $this->assertInstanceOf(
+            \DateTimeImmutable::class,
+            $shareResourceExpirationDateTime,
+            "Expected class to be " . \DateTimeImmutable::class
+            . " but found " . ($shareResourceExpirationDateTime !== null ? get_class($shareResourceExpirationDateTime) : "null")
+        );
```

Comment:
what about simplifying it like that

```suggestion
        $this->assertInstanceOf(
            \DateTimeImmutable::class,
            $shareResourceExpirationDateTime,
            "Expected class to be 'DateTimeImmutable'"
            . " but found " . print_r($shareResourceExpirationDateTime, true)
        );
```

Reply:
there are more cases like that, e.g. https://github.com/owncloud/ocis-php-sdk/pull/186/files#diff-63ac34bf34e0074dd6979a9dd8623738a73f3d9f457baee479282eb29c1353acR156-R161